frame/nomination-pools/src/lib.rs

```diff
@@ -1396,6 +1584,12 @@ pub mod pallet {
 		PoolSlashed { pool_id: PoolId, balance: BalanceOf<T> },
 		/// The unbond pool at `era` of pool `pool_id` has been slashed to `balance`.
 		UnbondingPoolSlashed { pool_id: PoolId, era: EraIndex, balance: BalanceOf<T> },
+		/// A pool's commission payee has been changed.
+		PoolCommissionPayeeChanged { pool_id: PoolId, payee: T::AccountId },
+		/// A pool's commission setting has been changed.
+		PoolCommissionUpdated { pool_id: PoolId, commission: Perbill, payee: T::AccountId },
```

Comment:
```suggestion
		/// A pool's commission setting has been changed.
		PoolCommissionUpdated { pool_id: PoolId, commission: Perbill, payee: T::AccountId },
```

I think these two are the same event. Since we grouped commission and the account together, it also needs just one event.